src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -7130,4 +7130,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_ScriptsAndSubmissionsCannotHaveRequiredMembers" xml:space="preserve">
     <value>Required members are not allowed on the top level of a script or submission.</value>
   </data>
+  <data name="ERR_BadAbstractEqualityOperatorSignature" xml:space="preserve">
+    <value>One of the parameters of an equality, or inequality operator declared in interfaces must be a type parameter on the enclosing type constrained to the enclosing type</value>
```

Comment:
```suggestion
    <value>One of the parameters of an equality, or inequality operator declared in interfaces must be a type parameter on {0} constrained to {0}</value>
```

Consider putting the actual type in the error message, I think it will make it clearer.